### ISSUE TITLE
mp4dash HLS not referencing I-Frame playlist?

### ISSUE THREAD

**Issue by richardbushell** [axiomatic-systems/Bento4#388]
Hi

In testing the --hls  option using mp4dash, I note that the master.m3u8 playlist contains:-

```
# Audio
#EXT-X-MEDIA:TYPE=AUDIO,GROUP-ID="audio/mp4a",LANGUAGE="en",NAME="English",AUTOSELECT=YES,DEFAULT=YES,URI="audio/en/mp4a/media.m3u8"

# Video
#EXT-X-STREAM-INF:AUDIO="audio/mp4a",AVERAGE-BANDWIDTH=363783,BANDWIDTH=392951,CODECS="avc1.64002A,mp4a.40.2",RESOLUTION=768x432
video/avc1/1/media.m3u8
...
#EXT-X-STREAM-INF:AUDIO="audio/mp4a",AVERAGE-BANDWIDTH=2729394,BANDWIDTH=2945641,CODECS="avc1.64002A,mp4a.40.2",RESOLUTION=1280x720
video/avc1/5/media.m3u8

# I-Frame Playlists
#EXT-X-I-FRAME-STREAM-INF:AVERAGE-BANDWIDTH=265686,BANDWIDTH=287585,CODECS="avc1.64002A",RESOLUTION=768x432,URI="video/avc1/1/media.m3u8"
...
#EXT-X-I-FRAME-STREAM-INF:AVERAGE-BANDWIDTH=2631297,BANDWIDTH=2840275,CODECS="avc1.64002A",RESOLUTION=1280x720,URI="video/avc1/5/media.m3u8"
```
But mp4dash also creates files named iframes.m3u8 which are NOT referenced in this master.m3u8 playlist.

Should the entire # I-Frame Playlists section be REMOVED from the master.m3u8 file generated, or should the URIs within the I-Frames Playlist section be iframes.m3u8 rather than media.m3u8 ???

The iframes.m3u8 files seem unreferenced and unused?

Please advise.

**Comment by richardbushell** [comment]
I have issued a Pull Request to fix the iframes playlist path, here:
https://github.com/axiomatic-systems/Bento4/pull/389

**Comment by richardbushell** [comment]
Can my PR be pulled in to fix this please?

**Comment by barbibulle** [comment]
Fixed by merging PR #389
